www/lobby/id.ts

```diff
@@ -1,15 +1,38 @@
 import { Router, Request, Response } from 'express';
 import { User } from '../../lib';
 import { Lobby } from '../../lib/lobby';
+import { validateJwt, validateLobbyJwt } from '../../utils/jwt';
 
 export const lobby_id_router = Router();
 
 // Join a lobby
 /**
  * Body Params: id, lobbyToken, refreshToken
  */
-lobby_id_router.post('/:id', async (req: Request, res: Response) => {
-  res.status(200).json('placeholder post');
+lobby_id_router.post('/:id', validateJwt, validateLobbyJwt, async (req: Request, res: Response) => {
+  const userId: string = req.body.id;
+  const lobbyId: string = req.params.id;
+  const decodedLobbyId: string | undefined = req.body.lobbyId;
+
+  //check if user and lobby id's are valid
+  const verified = await verifyIds(userId, lobbyId);
+  if (!verified) return res.status(404).json('User or Lobby not found in database').end();
+  const lobby = verified[1];
```

Comment:
what if we make the verifyIds function return an object? IMO it looks cleaner?

i.e. 

```ts
const {lobby} = verified;
```

lmk ur thoughts!

Reply:
Ok sounds good!